spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -3,61 +3,113 @@
 require_relative '../../../lib/reek/spec'
 
 RSpec.describe Reek::Spec::ShouldReekOf do
-  context 'rdoc demo example' do
-    let(:ruby) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
+  describe 'smell type check' do
```

Comment:
I refactored the whole file so the diff is a little messy I'm afraid. Probably better if you check the whole file when reviewing, not the diff.